src/templates/dataGridTable.jsx

```diff
@@ -95,7 +95,7 @@ class DataGridTable extends React.PureComponent {
         const containerClasses = ClassNames('ui', `${classNamePrefix}_table`, className);
         const bodyClasses = ClassNames({ [`${classNamePrefix}_drop_shadow`]: dropShadow });
         const isSelectable =
-            _.isFunction(rowProps) && _.isFunction(rowProps().onClick);
+            !_.isUndefined(rowProps) && _.isFunction(rowProps().onClick);
```

Comment:
Just curious: Any particular reason for this change.  Since it should either be a function or `undefined`, the prior version should have been okay too, right?